deepchecks/nlp/metric_utils/token_classification.py

```diff
@@ -0,0 +1,230 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Utils module containing scorers for computing token classification metrics."""
+import typing as t
+from collections.abc import Sequence
+
+import numpy as np
+from seqeval.metrics import accuracy_score, f1_score, precision_score, recall_score
+from seqeval.scheme import Token
+from sklearn.metrics import make_scorer
+
+from deepchecks.core.errors import DeepchecksValueError
+from deepchecks.nlp.text_data import TTokenLabel
+
+__all__ = ['get_default_token_scorers', 'validate_scorers', 'get_scorer_dict']
+
+DEFAULT_AVG_SCORER_NAMES = ('f1_macro', 'recall_macro', 'precision_macro')
+DEFAULT_PER_CLASS_SCORER_NAMES = ('f1_per_class', 'f1_per_class', 'f1_per_class')
+
+
+class SpanAligner:
+    """An object for transforming the labels and predictions of a specific dataset.
+
+    Labels come in the deepchecks accepted format and are returned as a list of lists of string annotations.
```

Comment:
Can add more detail how the output format should look like. maybe add an example